jxl_cli/src/bin/jxlinspect.rs

```diff
@@ -127,7 +129,42 @@ fn parse_jxl_codestream(data: &[u8], verbose: bool) -> Result<(), jxl::error::Er
         println!("ICC profile length: {} bytes", icc_data.len());
     }
     // TODO(firsching): add frame header parsing for each frame
-
+    if file_header.image_metadata.animation.is_some() {
```

Comment:
I assume the spec only allows multiple frames if `image_metadata.animation.is_some()`, but since the frame header has an `is_last` field, wouldn't it be more defensive to the presence of more frame headers as the criteria for showing multiple frame headers? That way a broken jxl image (with multiple frame headers but no animation set) would still be more easily debuggable using jxlinspect. WDYT?

Reply:
no, the spec also allows multiple frame if it is not an animation.
I guess we have to decide what info about frames we want to display in general...